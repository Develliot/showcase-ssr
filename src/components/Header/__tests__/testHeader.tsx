import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Header } from '../';

// not much point testing not visible because snapshot
// won't capture changes in styled component
it('renders visible', () => {
    const wrapper: ShallowWrapper = shallow(
        <Header currentRoute='/' visible={true} />
    );
    expect(wrapper).toMatchSnapshot();
});
