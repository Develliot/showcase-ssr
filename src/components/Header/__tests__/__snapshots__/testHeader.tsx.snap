// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders visible 1`] = `
<styles__HeaderWrapper
  visible={true}
>
  <styles__HeaderInner>
    <Link
      href="/"
      passHref={true}
    >
      <styles__LogoLink
        tabIndex={0}
        title="link to home"
      >
        <Typography__H2
          color="white"
        >
          Showcase
        </Typography__H2>
        <Layout__VerticalSpacer
          size="medium"
        />
      </styles__LogoLink>
    </Link>
    <nav>
      <styles__Links>
        <HeaderNavLink
          selected={true}
          title="Home"
          to="/"
        />
        <HeaderNavLink
          selected={false}
          title="Design System"
          to="/designSystem"
        />
        <HeaderNavLink
          selected={false}
          title="Using APIs"
          to="/usingAPIs"
        />
      </styles__Links>
    </nav>
  </styles__HeaderInner>
</styles__HeaderWrapper>
`;
